index.ejs

```diff
@@ -2,14 +2,81 @@
 <html xmlns='http://www.w3.org/1999/xhtml'>
 
 <head>
-  <meta http-equiv="content-type" content="application/vnd.hbbtv.xhtml+xml; charset=utf-8">
+  <meta http-equiv='content-type' content='application/vnd.hbbtv.xhtml+xml; charset=utf-8'>
   </meta>
   <title>Consent Management Test</title>
-  <script type="text/javascript" src="<%-HTTP_PROTOCOL%>://<%-CONSENT_SERVER_HOST%>/<%-API_VERSION%>/loader-with-banner.js?channelId=3100"></script>
+  <script type='text/javascript'
```

Comment:
would be great if the exposed function would match the name of the included script, e.g. `__tcfapi` is provided thorugh `tcfapi.js`